pySDC/projects/PinTSimE/estimation_check_implicit.py

```diff
@@ -0,0 +1,318 @@
+import numpy as np
```

Comment:
So, this file looks a lot like the original `estimation_check`. Can't you combine them and re-use what you already have? If this only changes the treatement of the ODE system, but not parameters, output, plot etc, you can make this more abstract.